gitapp/lib/users_list/users_list_presenter.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:gitapp/users_list/api_repository.dart';
+import 'package:gitapp/users_list/user_model.dart';
+
+class UsersList {
```

Comment:
Why in `users_list_presenter` we have class `UserList` ?

Reply:
Solved